installation/multi_node_debian.rst

```diff
@@ -59,15 +59,21 @@ Before starting the database let's change its access permissions. By default the
 
 .. note::
   Your DNS settings may differ. Also these settings are too permissive for some environments, see our notes about :ref:`worker_security`. The PostgreSQL manual `explains how <http://www.postgresql.org/docs/current/static/auth-pg-hba-conf.html>`_ to make them more restrictive.
+ Based on your DNS you can add another line with your IP address to allow the host unrestricted access, if you do not have an IP address that begins with 10.*.*.*. Otherwise add the line given for IPV4 address space. 
 
 **4. Start database servers, create Citus extension**
 
 ::
 
   # start the db server
   sudo service postgresql restart
+   or 
+  sudo systemctl restart postgresql@14-main
   # and make it start automatically when computer does
   sudo update-rc.d postgresql enable
+  
+.. note:: 
+Ensure you are starting the right postges version. You can check this by running ``ps -ef | grep Postgres``
```

Comment:
Could the `ps -ef ...` be woven into the restart commands above? Ideally we could have the script itself detect the right service to restart.

Reply:
Yes, that would be the best thing to do to ensure users are starting the right thing automatically.